### DISCUSSION TITLE
ansible practice

### DISCUSSION THREAD

**Discussion by ganeshkumardesai** [learndevopsonline/learndevopsonline#1867] [Q&A] [answered]
my sql server not requested to 
![Screenshot 2024-03-21 135125](https://github.com/learndevopsonline/learndevopsonline/assets/161687860/232fa0e1-5250-4c4a-9393-f9c5010c4057)
host

**Comment by chaithanya1812** [comment]
Hi @ganeshkumardesai,
## can you post the error like this way, below I tagged

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/61d8028f-dff6-4863-bd62-e9eb2cd6f39d)

**Comment by chaithanya1812** [comment]
# regarding to issue, service name of mysql is mysqld
## name: mysqld
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/7e1e128a-0e37-46c7-a4db-c981296cbecb)

**Comment by ganeshkumardesai** [comment]
even i had tried to change with mysql to mysqld still i am getting error
![Screenshot 2024-03-21 163154](https://github.com/learndevopsonline/learndevopsonline/assets/161687860/9a35e323-ea0e-485f-85f4-7f521369c33f)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/161687860/8c5c3bb1-f77b-4c93-9064-e7bdba184e1f)

**Comment by chaithanya1812** [comment]
## name: mysql-server
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/1beb5c08-2615-4353-8afe-adbfb57bfdb2)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/34214406-ca1d-401b-8b67-988fb3fa9d96)

**Comment by ganeshkumardesai** [comment]
i am getting error here
![image](https://github.com/learndevopsonline/learndevopsonline/assets/161687860/9898666d-ae51-43e1-b981-85bac1d09c9d)

**Comment by chaithanya1812** [comment]
## The error was expected, Ignore this, 
## focus on  other components, after completion of upcoming videos you will get know..

**Reply by ganeshkumardesai** [reply]
okay thank you

**Comment by ganeshkumardesai** [comment]
hii good morning while i am trying to make ansible pull i am getting error like these
![image](https://github.com/learndevopsonline/learndevopsonline/assets/161687860/f15ec4c9-f871-4e6a-961f-eb44380d5791)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/161687860/29d439ea-0696-4914-836d-5a0293e7b73f)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/161687860/f9694157-3f37-46d2-a8ae-723c6175ee66)

**Comment by chaithanya1812** [comment, marked as answer]
it's not -u
It's   -U ( capital letter)

**Reply by ganeshkumardesai** [reply]
THANKS CHAITANYA I GOT IT

**Comment by ganeshkumardesai** [comment]
can u help me out of these
![image](https://github.com/learndevopsonline/learndevopsonline/assets/161687860/94e42c77-d1eb-4740-b018-16f17ebd4754)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/161687860/47998fca-e389-43e6-a494-f1aa606aec96)

**Comment by chaithanya1812** [comment]
Hi @ganeshkumardesai,  In modules directory we need to write resources block,  get out of module directory.
Outside create main.tf file & Call the module,
In words I can't explain for more  use the lab query session

**Reply by ganeshkumardesai** [reply]
yes i got it chaitanya thank you

**Comment by ganeshkumardesai** [comment]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/161687860/26ffdcf3-731b-4673-9397-8fa32481d9db)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/161687860/e560752d-825a-4049-bcae-f95182e5c3a5)

hii chaitanya while i am trying to terraform apply i am getting error can u help me out!

**Comment by chaithanya1812** [comment]
Hi @ganeshkumardesai,
## Issue :- In your code no where I can't find this  block of code !..

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/ee430892-7e7a-496f-a591-3dc53588332f)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/c7c9d9b1-5847-4ecb-a1dc-e37c5b55e7b7)

**Comment by chaithanya1812** [comment]
Hi @ganeshkumardesai, 
## Next time, when you a got error, create a new discussion item..

**Reply by ganeshkumardesai** [reply]
okay

**Comment by ganeshkumardesai** [comment]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/161687860/9a99e30c-e97e-4613-b1e6-463f9715ff45)

hii chaitanya.Still i am getting error like these..

**Comment by chaithanya1812** [comment]
## In your run.sh, it should be like
  -var-file=< file path >
